django-stubs/contrib/sessions/base_session.pyi

```diff
@@ -11,9 +11,9 @@ class BaseSessionManager(models.Manager[_T]):
     def save(self, session_key: str, session_dict: dict[str, Any], expire_date: datetime) -> _T: ...
 
 class AbstractBaseSession(models.Model):
-    expire_date: datetime
-    session_data: str
-    session_key: str
+    session_key: models.CharField
+    session_data: models.TextField
+    expire_date: models.DateTimeField
     objects: Any
```

Comment:
We should add type parameters to the fields and while we're at it also set a type for `objects`.

e.g. `session_key: models.CharField[str, str]`

Looks like `objects` should have: `objects: ClassVar[BaseSessionManager[Self]]`

Though it looks like fields on other models in the stubs have been declared as runtime expressions. So let's do that here too for the sake of consistency

```suggestion
    session_key = models.CharField(max_length=40, primary_key=True)
    session_data = models.TextField()
    expire_date = models.DateTimeField()
    objects: ClassVar[BaseSessionManager[Self]]
```

Reply:
While looking at the runtime, I don't think `objects` should be removed: https://github.com/django/django/blob/bcbc4b9b8a4a47c8e045b060a9860a5c038192de/django/contrib/sessions/base_session.py#L32